src/View.Uno/Controls/SwipeRefresh/SwipeRefresh.Android.cs

```diff
@@ -80,6 +81,21 @@ protected override void OnContentChanged(object oldValue, object newValue)
 			{
 				_nativeSwipeRefresh.Content = Content as Android.Views.View;
 			}
+
+			if (Content is BindableView bv)
+			{
+				bv.DataContext = DataContext;
+			}
+		}
+
+		protected override void OnDataContextChanged()
+		{
+			base.OnDataContextChanged();
+
+			if (Content is BindableView bv)
+			{
+				bv.DataContext = DataContext;
```

Comment:
Don't we need to do that on iOS as well?

Reply:
This was an Android only issue. iOS was propagated properly